### PR TITLE
Merge Readthedocs into Main

### PR DESCRIPTION
**Pull request by RainierBarrett**
So we can check doc builds as we continue on other branches

### CODE REVIEW

**Comment by chrisjonesBSU** [comment]
I think we probably don't need the utils directory part of the documentation right now. Most of that is used internally, and at some point we should probably move the `actions.py` stuff into its own module.  

Other than that, I think we can merge this.

**Review by marjanAlbouye — APPROVED** [review]
@RainierBarrett 
We can merge this to main if things are ready. I will update my docstrings branch once this is merged.

**Pull request closed without merging**
